yocto-base/01_base_os.robot

```diff
@@ -0,0 +1,59 @@
+*** Settings ***
+Library             Collections
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../../sonoff-rest-api/sonoff-api.robot
+Resource            ../../rtectrl-rest-api/rtectrl.robot
+Resource            ../../variables.robot
+Resource            ../../keywords.robot
+Resource            ../lib/sd-wire.robot
+Resource            ../lib/linux.robot
+
+Suite Setup         Prepare Test Suite
+Suite Teardown      Log Out And Close Connection
+
+
+*** Test Cases ***
+Boot
+    [Documentation]    This test aims to see whether the device can boot.
+    Variable Should Exist    ${DUT_PASSWORD}
+    Sonoff Power Off
+    Sleep    5s
+    Sonoff Power On
+    Serial Root Login Linux    ${DUT_PASSWORD}
+    ${output}=    Telnet.Execute Command    echo hi
+    ${output}=    Get Line    ${output}    0
+    Should Be Equal As Strings    ${output}    hi
```

Comment:
@WiktorG351 what other methods would be available to verify if the platform booted up? @TomaszAIR cc
I think about something like a sign of life, but with Yocto/U-Boot elements. On the other hand, logging into the system is enough to say, that the system works (boot).

Reply:
@WiktorG351 @artur-rs Can't we determine the fact that the platform is booted by simply logging into the system?

Reply:
@TomaszAIR okay, i removed the reduntant echo hi checks [here](https://github.com/Dasharo/open-source-firmware-validation/pull/34/commits/9c0942bf906963d4e4246f5650ff31d3623ce7be)